scripts/src/internal/services.ts

```diff
@@ -30,7 +30,6 @@ export type WalletCreationSuccessData = {
 export async function walletCreationSuccess(data: WalletCreationSuccessData) {
 	createStellarAccountCreationSucceeded(data.id).report();
 	logger().info("wallet created", data);
-	await updateWalletCreationDate(data.wallet_address, "createdDateKin2");
```

Comment:
This location was wrong for this update. The callback is called from both kin2 and kin3 payment service and it always updates kin2 creation date.
I change it to update the creation date to just before requesting the createWallet request on either kin2 or kin3.

Reply:
ok